packages/core/useElementSize/index.ts

```diff
@@ -0,0 +1,39 @@
+import { MaybeRef } from '@vueuse/shared'
+import { ref } from 'vue-demi'
+import { ResizeObserverOptions, useResizeObserver } from '../useResizeObserver'
+
+export interface ElementSize {
+  width: number
+  height: number
+}
+
+/**
+ * Reactive size of an HTML element.
+ *
+ * @see   {@link https://vueuse.js.org/useElementSize}
+ * @param target
+ * @param callback
+ * @param options
+ */
+export function useElementSize(
+  target: MaybeRef<Element | null | undefined>,
+  initialSize: Size = { width: Infinity, height: Infinity },
```

Comment:
```suggestion
  initialSize: ElementSize = { width: 0, height: 0 },
```

Making the default size to zero makes more sense to me. (just like an empty <div/>)